docs/source/contributing/docker_container.md

```diff
@@ -0,0 +1,21 @@
+(pr_checklist)=
```

Comment:
This is defining a target to which we can then link from anywhere in the docs to generate a link that points here. The `pr_checklist` label is already in use so it can't be used and it should be something specific to this file

Reply:
Hi, got it.